src/feverous/baseline/drqascripts/build_tfidf.py

```diff
@@ -75,7 +80,12 @@ def count(ngram, hash_size, doc_id):
 
     # Return in sparse matrix data format.
     row.extend(counts.keys())
-    col.extend([DOC2IDX[doc_id]] * len(counts))
+    print(f"DOC2IDX: {DOC2IDX}")
+    if doc_id in DOC2IDX:
+        col.extend([DOC2IDX[doc_id]] * len(counts))
+    else:
+        print(f"Warning: {doc_id} not found in DOC2IDX")
```

Comment:
In principle, all documents should be found in DOC2IDX. So if this error is thrown, it means that something fishy is going on. Maybe the warning can be extended to signify the urgency of this message.